components/style/workqueue.rs

```diff
@@ -276,21 +277,42 @@ impl<QueueData: Sync, WorkData: Send> WorkQueue<QueueData, WorkData> {
         }
 
         // Spawn threads.
+        let mut thread_handles = vec![];
```

Comment:
I use to prefer `vec![]` because it mimics the style of arrays and slices (with the square brackets). But yeah, you can use whatever you want.